lib/y2logsstep.pm

```diff
@@ -223,6 +223,18 @@ sub verify_license_has_to_be_accepted {
     }
 }
 
+sub verify_translation {
+    if (!check_var('VIDEOMODE', 'text')) {
```

Comment:
What's about checking this variable before method is called, or 
```perl
return if check_var('VIDEOMODE', 'text');
``` to avoid additional indent level.